package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Teamleader <development@teamleader.eu>",
   "betterScripts": {
     "compile": {
-      "command": "node bin/compile",
+      "command": "build-storybook -o dist",
```

Comment:
`dist` used to be our compiled version of the components. Now it's a compiled version of the storybooks? I guess that's now what we want to ship.

Reply:
I was just checking again, and I'm a bit confused. We have a gulpfile as well which copies stuff to `lib`? I think I'm always publishing just by running `npm run deploy:prod`. 
@nickwaelkens do you remember this?

Reply:
We really need to start documenting better 😢

Reply:
I do remember we switched to an extremely simple gulpfile to speed up our build process. Webpack was jumping through all kinds of hoops, because it's a copy/paste of the setup for marketplace/core/etc. Documentation is something we need to work on, I added it to my todo list!